templates/vue/src/utils.js

```diff
@@ -0,0 +1,71 @@
+import { connect, Contract, keyStores, WalletConnection } from 'near-api-js'
```

Comment:
This `utils.js` file is not needed, is it? It should be a symlink I think

Reply:
If you soft-symlink that file you'll get a vue compiler error:
```
> cd templates/vue/src
> mv utils.js NOSYMLINK-utils.js
> ln -s ../../../common/frontend/utils.js
> cd ..
> npm run serve

ERROR  Failed to compile with 1 errors                                                                                                                                                                                            
This dependency was not found:
* config in /home/lucio/repos/create-near-app/common/frontend/utils.js
```
Since config is now local to the template and `utils.js` does `import getConfig from './config'`, the compiler tries to look for 'config.js' in the same dir as 'utils.js', that's it: '/common/frontend/'

One solution is to *hard* symlink `utils.js`, but that will mess up git. 
The second solution is to have a copy of both files in the template: utils.js and config.js

Reply:
Interesting! Parcel has no problem with this in the `templates/vanilla` or `templates/react` directories when I test on my machine.

The duplication of this file makes me wonder if we should stop putting it in `common/frontend`, as we did with `config.js` 🤔

Reply:
I agree with you, we should stop putting utils.js in common/frontend